rasa_core/config.py

```diff
@@ -3,14 +3,14 @@
 from typing import Optional, Text, List
 
 from rasa_core import utils
-from rasa_core.policies import PolicyEnsemble
```

Comment:
I guess you moved these to make the commandline faster. 

Does it make sense to add a test to travis that checks how long it takes for the cli to start?

Reply:
added reminder to stack pr